recipe-app/src/CSS/styles.css

```diff
@@ -0,0 +1 @@
+@import url(Card.css);
```

Comment:
Neat work here, @karaguarraci! Thanks for introducing the `@import` CSS at-rule.

Reply:
No worries! :)